src/EnergyPlus/Plant/PlantAvailManager.hh

```diff
@@ -51,6 +51,24 @@
 namespace EnergyPlus {
 namespace DataPlant {
 
+    enum class SystemAvailabilityType
+    {
+        Invalid = -1,
+        Scheduled,
+        ScheduledOn,
+        ScheduledOff,
+        NightCycle,
+        DiffThermo,
+        HiTempTOff,
+        HiTempTOn,
+        LoTempTOff,
+        LoTempTOn,
+        NightVent,
+        HybridVent,
+        OptimumStart,
+        Num
+    };
+
```

Comment:
This enum class is being used in multiple files, so I moved it to a smaller `.hh` file (PlantAvailManager) to decrease the load being copied. Problem was that it changed to `DataPlant` namespace which might be confusing. Also, the advantage may disappear if `PlantAvailManager` grows in size at a later point of time.

Reply:
Slightly better than a bunch of `constexpr int`vars. 😄